companies/realtor.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I'm not sure how this file got included :( Can you remove it from this branch?

Reply:
Oh no, can I remove it by deleting the file?

Reply:
![Screen Shot 2020-10-25 at 12 28 07 PM](https://user-images.githubusercontent.com/42004648/97112951-98049800-16bd-11eb-91e1-730247673bc8.png)
Delete the file like this?

Reply:
Try this instead:

In your text editor, delete the file manually.
then `git add .`
`git commit -m "remove realtor.com"
`git push`

Hopefully that should work 🤞🏻